daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-features-concepts.md

```diff
@@ -151,6 +151,15 @@ Learn more about [external system interaction.]({{< ref "workflow-patterns.md#ex
 
 ## Limitations
 
+### Purging
+
+Workflow state can be purged from a state store, purging all its history and removing all metadata related to a specific workflow instance. The purge capability is typically used for workflows that have run to a `COMPLETED` or `TERMINATED` state. 
+
+There are other times you may choose to purge a workflow history and reuse a workflow instance with new inputs, like: 
+
+- `CONTINUED_AS_NEW` state
+- A `FAILED` workflow instance
```

Comment:
I don't really consider these points to be "limitations" and I don't recommend we put this information here. It's not something that someone learning about Dapr Workflow needs to be concerned with as they work through the high-level concepts. I think it would make more sense to mention in the API documentation that only completed, failed, and terminated workflows can be purged.

I also don't think we need to mention `CONTINUED_AS_NEW`. That's a very difference concept from "purge". Instead, I think we can update our existing discussion of continue-as-new (further up in this doc) to more clearly mention that the continue-as-new API will truncate the existing history, replacing it with a new history.

Reply:
Yes, this should be placed _before_ the limitations section, not in the limitations section. 

Agree with these changes. This is supposed to be an overall description of purge capability and when it should be used, as you describe. 

Yes, remove the CONTINUED_AS_NEW mention from here and fix the existing description in this topic.